components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -354,9 +354,7 @@ void AdsServiceImpl::OnGetDeviceId(std::string device_id) {
 }
 
 bool AdsServiceImpl::UserHasOptedIn() const {
-  return base::FeatureList::IsEnabled(
-             brave_news::features::kBraveNewsFeature) &&
```

Comment:
can we please remove `#include "brave/components/brave_news/common/features.h"` from `ads_service_impl.cc`, thanks